src/server/test/db/compressedReadingsTests.js

```diff
@@ -50,40 +44,40 @@ mocha.describe('Compressed Readings', () => {
 		const reading1 = new Reading(meter.id, 100, timestamp1, timestamp2);
 		const reading2 = new Reading(meter.id, 200, timestamp2, timestamp3);
 		const reading3 = new Reading(meter.id, 300, timestamp4, timestamp5);
-		await Reading.insertAll([reading1, reading2, reading3]);
+		await Reading.insertAll([reading1, reading2, reading3], conn);
 
 		// Compress the three points to two points.
-		const compressedReadings = await Reading.getCompressedReadings([meter.id], timestamp1, timestamp5, 2);
-		chai.expect(compressedReadings[meter.id]).to.have.lengthOf(2);
+		const compressedReadings = await Reading.getCompressedReadings([meter.id], timestamp1, timestamp5, 2, conn);
+		expect(compressedReadings[meter.id]).to.have.lengthOf(2);
 		const expectedFirstCompressedRate = ((reading1.reading) + (reading2.reading)) / 2;
-		chai.expect(compressedReadings[meter.id][0].reading_rate).to.be.closeTo(expectedFirstCompressedRate, 0.0001);
+		expect(compressedReadings[meter.id][0].reading_rate).to.be.closeTo(expectedFirstCompressedRate, 0.0001);
 		const expectedSecondCompressedRate = reading3.reading;
-		chai.expect(compressedReadings[meter.id][1].reading_rate).to.be.closeTo(expectedSecondCompressedRate, 0.0001);
+		expect(compressedReadings[meter.id][1].reading_rate).to.be.closeTo(expectedSecondCompressedRate, 0.0001);
 	});
 
 	mocha.it('compresses readings that overlap an end point', async () => {
 		const reading1 = new Reading(meter.id, 100, timestamp1, timestamp2);
 		const reading2 = new Reading(meter.id, 200, timestamp2, timestamp3);
-		await Reading.insertAll([reading1, reading2]);
+		await Reading.insertAll([reading1, reading2], conn);
```

Comment:
I don't know the logic but want to be sure conn is properly set in this code and next.